css/style.css

```diff
@@ -0,0 +1,430 @@
+body {
+  font-family: "Montserrat", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  max-width: 100%;
+}
+
+.section {
+  max-width: 1150px;
+  margin: 2px auto;
+  margin-bottom: 0;
+}
+/* HEADER */
+.header {
+  background: radial-gradient(rgb(194, 74, 126), rgb(105, 86, 235));
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 0;
+}
+
+.navbar__logo {
+  max-width: 100px;
+}
+
+.hero {
+  display: flex;
+  padding-top: 20px;
+  padding-bottom: 60px;
+}
+
+h1,
+a {
+  color: white;
+  text-decoration: none;
+}
+
+.hero__title {
+  flex-direction: column;
+}
+
+.hero__title h1 {
+  font-size: 2em;
+  font-family: "Montserrat";
+  font-weight: bold;
+  margin-bottom: 5px;
+}
+
+.hero__title p {
```

Comment:
Jeśli stylujemy rpzy pomocy BEM to każdy element powinien mieć klasę i do niej przypisujemy reguły :)